cmd/vulcan-wpscan/wpscan.go

```diff
@@ -26,11 +33,18 @@ var (
 	wpscanForceParams = []string{"--force", "--wp-content-dir", "wp-content"}
 	wpscanScopeParams = []string{"--scope"}
 	wpscanTokenParams = []string{"--api-token"}
+	wpscanUserAgent   = []string{"--user-agent", "Vulcan"}
+
+	// --ignore-main-redirect string matching list.
```

Comment:
I think it would be good to explain a bit the rationale for this change so in the future we know why is this here.

Reply:
Yup, good suggestion. Explanation added here [5e7a74d](https://github.com/adevinta/vulcan-checks/pull/182/commits/5e7a74d58e579d8cfcb685df7a5a480f3115bad4)